src/eppo_value.ts

```diff
@@ -56,18 +90,22 @@ export class EppoValue {
   }
 
   static Bool(value: boolean): EppoValue {
-    return new EppoValue(ValueType.BoolType, value, undefined, undefined);
+    return new EppoValue(ValueType.BoolType, value, undefined, undefined, undefined);
   }
 
   static Numeric(value: number): EppoValue {
-    return new EppoValue(ValueType.NumericType, undefined, value, undefined);
+    return new EppoValue(ValueType.NumericType, undefined, value, undefined, undefined);
   }
 
   static String(value: string): EppoValue {
-    return new EppoValue(ValueType.StringType, undefined, undefined, value);
+    return new EppoValue(ValueType.StringType, undefined, undefined, value, undefined);
+  }
+
+  static JSON(value: string, typedValue: object): EppoValue {
```

Comment:
Two arguments to instantiate the JSON value because it needs to be able to return both the string and object format